pkg/process/metadata/workloadmeta/extractor.go

```diff
@@ -52,12 +59,17 @@ func NewWorkloadMetaExtractor(config config.ConfigReader) *WorkloadMetaExtractor
 		cache:        make(map[string]*ProcessEntity),
 		cacheVersion: 0,
 		// Keep only the latest diff in memory in case there's no consumer for it
-		diffChan: make(chan *ProcessCacheDiff, 1),
+		diffChan:        make(chan *ProcessCacheDiff, 1),
+		cacheSizeGuage:  telemetry.NewGauge(subsystem, "cache_size", nil, "The cache size for the workloadMetaExtractor"),
+		oldDiffDropped:  telemetry.NewSimpleCounter(subsystem, "old_diff_dropped", "The number of times a diff is removed from the queue due to the diffChan being full."),
```

Comment:
Recognizing these are all mostly internal, but maybe just `diff_dropped` and `diff_chan_full`?

Reply:
In both cases there's a message being dropped, so maybe `old_diff_dropped` and `new_diff_dropped`? It's expected for old diffs to be dropped if there's no consumers but a diff recently produced should never be dropped.

Reply:
Tbh, @xornivore's names feel a bit clearer to me. I think in the future the difference between an old and a new diff won't be super clear, and the help text explains clearly that it is dropped. I'll go ahead and change it to those.